framework/src/bcs/DGFunctionDiffusionDirichletBC.C

```diff
@@ -42,7 +42,7 @@ DGFunctionDiffusionDirichletBC::DGFunctionDiffusionDirichletBC(const InputParame
 Real
 DGFunctionDiffusionDirichletBC::computeQpResidual()
 {
-  const unsigned int elem_b_order = static_cast<unsigned int> (_var.getOrder());
+  const unsigned int elem_b_order = _var.order();
```

Comment:
@jwpeterson - what are your thoughts on removing this explicit cast. `order()` returns an `Order`. We had an explicit cast but it appears @YaqiWang likes the implicit conversion. I think the explicit one is probably better, but I'm willing to listen to arguments.